File: app/src/purefloss/java/com/github/premnirmal/ticker/StocksApp.java
package com.github.premnirmal.ticker;

import android.support.multidex.MultiDexApplication;

import com.github.premnirmal.tickerwidget.R;

import uk.co.chrisjenx.calligraphy.CalligraphyConfig;

/**
 * Created by premnirmal on 12/21/14.
 */
public class StocksApp extends BaseApp {

    @Override
    public void onCreate() {
        super.onCreate();
        Injector.init(this);
    }

}

File: app/src/dev/kotlin/com/github/premnirmal/ticker/StocksApp.kt
package com.github.premnirmal.ticker

import timber.log.Timber

/**
 * Created by premnirmal on 2/28/16.
 */
class StocksApp : BaseApp() {

  override fun initCrashLogger() {
    Timber.plant(Timber.DebugTree())
  }
}